### PR TITLE
Update benchmark CI configuration

### PR DESCRIPTION
**Pull request by unexge**
---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and I agree to the terms of the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).

### CODE REVIEW

**Review by oleg-lvovitch-aws — DISMISSED** [review]
🚢

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         # Store the results and deploy GitHub pages automatically if the results are from main branch
         auto-push: ${{ inputs.publish }}
-        comment-on-alert: true
+        comment-on-alert: false
```
Comment (monthonk) [inline .github/workflows/bench.yml:81]: This is for throughput check. Do we also want to disable them?

Comment (unexge) [reply .github/workflows/bench.yml]: Reverted changes for throughput benchmarks with https://github.com/awslabs/mountpoint-s3/pull/1063/commits/df9d7c3953a5b21a272314b70cfff7fcd4470f24

**Review by monthonk — APPROVED** [review]
LGTM!

**Pull request merged**